build-rr.sh

```diff
@@ -200,7 +201,8 @@ checksubmodules ()
 	# We assume that if the git submodule command fails, it's because
 	# we're using external $RUMPSRC.
 	if git submodule status ${RUMPSRC} 2>/dev/null | grep -q '^-' \
-	    || git submodule status ${BUILDRUMP} 2>/dev/null | grep -q '^-';
+	    || git submodule status ${BUILDRUMP} 2>/dev/null | grep -q '^-' \
+	        || git submodule status ${SOLO5SRC} 2>/dev/null | grep -q '^-';
```

Comment:
nit: allignment

Reply:
ahh, that was on purpose:

```
        if git submodule status ${RUMPSRC} 2>/dev/null | grep -q '^-' \
            || git submodule status ${BUILDRUMP} 2>/dev/null | grep -q '^-' \
                || git submodule status ${SOLO5SRC} 2>/dev/null | grep -q '^-';
```
thought, this looked prettier than:
```
        if git submodule status ${RUMPSRC} 2>/dev/null | grep -q '^-' \
            || git submodule status ${BUILDRUMP} 2>/dev/null | grep -q '^-' \
            || git submodule status ${SOLO5SRC} 2>/dev/null | grep -q '^-';
```